pybandits/base.py

```diff
@@ -93,29 +89,31 @@ class BaseMab(PyBanditsBaseModel, ABC):
         The list of possible actions, and their associated Model.
     strategy: Strategy
         The strategy used to select actions.
-    epsilon: Optional[Float01], defaults to None
+    epsilon: Optional[Float01]
         The probability of selecting a random action.
+    default_action: Optional[ActionId]
+        Default action to select if the epsilon-greedy approach is used. None for random selection.
```

Comment:
Suggestion to make it more verbose:
```
The default action to select with a probability of epsilon when using the epsilon-greedy approach. If `default_action` is None, a random action from the action set will be selected with a probability of epsilon.
```

Reply:
Thanks. I also changed the docstring on smab.py and cmab.py.